lms/templates/course_modes/choose.html

```diff
@@ -73,7 +73,13 @@ <h3 class="title">${_("Sorry, there was an error when trying to enroll you")}</h
                 <article class="register-choose content-main">
                     <header class="page-header content-main">
                         <h3 class="title">
-                            ${_("Congratulations!  You are now enrolled in {course_name}").format(course_name=course_name)}
+                            % if show_enterprise_context:
+                                ${_("Welcome, {username}! You are about to enroll in {course_name}, from "
```

Comment:
@griffresch here is the new version of the Track Selection message -- please confirm when you have a chance, thank you.